recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -1,81 +1,75 @@
 SUMMARY = "WPE Framework common plugins"
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=39fb5e7bc6aded7b6d2a5f5aa553425f"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=47b1321b4411e96bb5aeb94521850d43"
 PR = "r1"
 
 require include/wpeframework-plugins.inc
 
-SRC_URI = "git://github.com/rdkcentral/ThunderNanoServices.git;protocol=git;branch=master \
+SRC_URI = "\
+git://github.com/rdkcentral/ThunderNanoServices.git;protocol=git;branch=master \
     file://index.html \
     file://osmc-devinput-remote.json \
-    file://0001-westeros-preload-libwesteros_gl.so.0.0.0.patch"
+    file://0001-westeros-preload-libwesteros_gl.so.0.0.0.patch \
+"
 
-SRCREV = "e7da86b0bf14459690f95dcd91a19f9a6145743c"
+SRCREV = "4fb9cd13bfb4a953397003a6eed6582e1a4a14e0"
 # ----------------------------------------------------------------------------
 
 # More complicated plugins are moved seperate includes
```

Comment:
I believe that's not needed anymore. Maybe a comment that here goes the plugin configuration